packages/psd/src/classes/Group.ts

```diff
@@ -16,15 +16,15 @@ export class Group implements NodeBase<NodeParent, NodeChild> {
 
   /** @internal */
   constructor(
-    private layerFrame: GroupFrame,
+    private layerFrame: GroupFrame | undefined,
```

Comment:
This is very weird to me, but there is one sample file where rendering layer fails here with:
```

TypeError: Cannot read properties of undefined (reading 'layerProperties')
    at get opacity [as opacity] (file://./packages/psd/dist/index.js:1:186)
    at get composedOpacity [as composedOpacity] (file://./packages/psd/dist/index.js:1:273)
    at get composedOpacity [as composedOpacity] (file://./packages/psd/dist/index.js:1:78558)
    at BI.composite (file://./packages/psd/dist/index.js:1:78007)
    at async file://./packages/psd/scripts/parse.mjs:113:7
    at async Promise.all (index 256)
    at async file://./packages/psd/scripts/parse.mjs:109:3
```

where parse.mjs is
```ts
import * as fs from "fs";

import Psd from "../dist/index.js";

for (const input of process.argv.slice(2)) {
  console.log(input);
  const arrayBuffer = fs.readFileSync(input);

  const psd = Psd.parse(arrayBuffer.buffer);

  await Promise.all(
    psd.layers.map(async (layer) => {
      await layer.userMask();
      await layer.realUserMask();
      await layer.composite();
    })
  );
}
```

thus the workaround.